test/runtests.jl

```diff
@@ -650,113 +650,12 @@ let callsites = find_callsites_by_ftt(issue152_another, (Tuple{Float64,Vararg{Fl
     @test !isempty(callsites)
 end
 
-# NOTE setup to for `cthulhu_ast`
+# NOTE setup too for `cthulhu_ast`
 include("sandbox.jl")
 using .CthulhuTestSandbox
 Revise.track(CthulhuTestSandbox, normpath(@__DIR__, "sandbox.jl"))
 
```

Comment:
Let's move these lines into test/codeview.jl as well, they're only used by the printer test anyway.

Reply:
Done.